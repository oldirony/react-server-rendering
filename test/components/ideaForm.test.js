import React from 'react';
import IdeaForm from '../../src/components/idea-form/idea-form';
import {shallow} from 'enzyme';

const ideaFormProps = {
	formAction: ()=>true,
	formTitle: 'My title',
	label: 'Submit',
	redirectTo: 'test',
	handleSubmit: func=>func
}

test('Base rendering', ()=>{
	const ideaForm = shallow(<IdeaForm {...ideaFormProps}/>);
	expect(ideaForm).toMatchSnapshot();
});