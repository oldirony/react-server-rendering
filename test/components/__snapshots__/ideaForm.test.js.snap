// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base rendering 1`] = `
<form
  onSubmit={[Function]}
>
  <h4
    className="title"
  >
    My title
  </h4>
  <div
    className="row"
  >
    <Field
      className="input inputTitle"
      component={[Function]}
      name="title"
      placeholder="Title here"
      type="text"
      validate={
        Array [
          [Function],
        ]
      }
    />
  </div>
  <div
    className="row"
  >
    <Field
      className="textarea mainTextarea"
      component="textarea"
      name="content"
      placeholder="Content here"
    />
  </div>
  <div
    className="section"
  >
    <h5
      className="subtitle"
    >
      Sections
    </h5>
    <FieldArray
      className="row"
      component={[Function]}
      name="sections"
    />
  </div>
  <div
    className="section"
  >
    <button
      className="button"
      type="submit"
    >
      Submit
    </button>
  </div>
</form>
`;
